app/src/androidTest/java/com/example/evently/ViewNotificationsTest.java

```diff
@@ -0,0 +1,251 @@
+package com.example.evently;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.concurrent.ExecutionException;
+
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+
+import com.google.android.gms.tasks.Tasks;
+import com.google.firebase.Timestamp;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.example.evently.data.EventsDB;
+import com.example.evently.data.NotificationDB;
+import com.example.evently.data.model.Category;
+import com.example.evently.data.model.Event;
+import com.example.evently.data.model.Notification;
+import com.example.evently.ui.entrant.ViewNotificationsFragment;
+
+@RunWith(AndroidJUnit4.class)
+public class ViewNotificationsTest extends EmulatedFragmentTest<ViewNotificationsFragment> {
```

Comment:
Example of a full UI test utilizing the existing infra. See this and base your tests off of it.

Notice how I extend `EmulatedFragmentTest` with a type parameter that specifies which fragment is being tested.